app/src/main/java/com/hedvig/app/service/push/PushNotificationWorker.kt

```diff
@@ -28,7 +30,9 @@ class PushNotificationWorker(
         if (!hasHedvigToken()) {
             return Result.retry()
         }
-        registerPushToken(pushToken)
+        CoroutineScope(IO).launch {
```

Comment:
This should probably be scoped to the worker - take a look at `CoroutineWorker` from `work-runtime-ktx`